pkg/generator/pool_test.go

```diff
@@ -24,6 +23,7 @@ func TestGetNow(t *testing.T) {
 		if pool.CurrentSize() == 5 {
 			break
 		}
+		runtime.Gosched()
```

Comment:
Can we put a short explanation of why we need to do that?

Reply:
98122675e077323612fb8120cd3b360e1398d6db